src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  color: $main-text-color;
+  background-color: #fff;
+  border: 1px solid $light-grey-color;
+  padding: 32px 16px 16px;
+  transition: $hover-duration;
```

Comment:
The variable `$hover-duration` is used for the transition property. Ensure that this variable is defined in your SCSS variables file to avoid compilation errors.